.github/workflows/ci-linux-on-main-pr.yml

```
@@ -26,15 +26,15 @@ jobs:
         run: |
           sudo apt-get update -y
           sudo apt-get install -y 
-          sudo apt-get remove -y llvm-12 llvm-11 clang-11 clang-format-11 clang-tidy-11 clang-format-14 clang-tidy-14 
+          sudo apt-get remove -y llvm-14 llvm-12 llvm-11 clang-11 clang-format-11 clang-tidy-11 clang-format-14 clang-tidy-14 
```

Consider grouping the versions together:
```suggestion
          sudo apt-get remove -y llvm-14 clang-format-14 clang-tidy-14 llvm-12 llvm-11 clang-11 clang-format-11 clang-tidy-11
```